src/Sylius/Bundle/ShopBundle/Resources/private/entry.js

```diff
@@ -0,0 +1,15 @@
+import './js/shim/shim-jquery';
+import './js/shim/shim-lightbox';
+import './js/shim/shim-semantic-ui';
+
+import 'semantic-ui-css/semantic.css';
+import 'lightbox2/dist/css/lightbox.min.css';
+
+import '../../../UiBundle/Resources/private/js/app';
```

Comment:
Could we pass `uiBundlePath` variable from `webpack.config.js` here? Going outside of the current bundle seems a bit like crossing some boundary.

I don't hold this opinion strongly, but I want to discuss it first :)

Reply:
in the `webpack.config.js` file we have the alias `sylius/ui` which is necessary for the `app.js` files. We can create another alias, e.g. `sylius/ui_resources` and then we'll be able to use it instead of `../../../UiBundle/Resources/private`

Reply:
Sounds good! This way we'll get the absolute path to the resources. What is the naming convention here, should it be `sylius/ui_resources` or `sylius/ui-resources`?

Reply:
imo `sylius/ui-resources`